mognet/model/result.py

```diff
@@ -66,7 +66,7 @@ class ResultValueHolder(BaseModel):
     """
 
     value_type: str
-    raw_value: Any
+    raw_value: Any = None
```

Comment:
Not sure I agree with these defaults. It doesn't make sense (to me) to construct a `ResultValueHolder` with an unset `raw_value`, even if it defaults to `None`.

Personally, I would remove these defaults and pass the values explicitly on the model constructors.

The same on the others.

PS: If I left it like that when I wrote the original version, I'm sorry. I have since changed my point of view on this matter :)